sesman.el

```diff
@@ -72,6 +72,14 @@ sessions running in dependent projects."
   :type 'boolean
   :package-version '(sesman . "0.3.2"))
 
+(defcustom sesman-follow-symlinks t
+  "If non-nil expand file paths and follow symlinks using `file-truename'.
```

Comment:
I am a bit uncomfortable that this goes against the default value of `find-file-visit-truename`. What are your thought regarding this?

Reply:
@vspinu  Agree. But it will be confusing either way. When you use something like `ranger` it suppresses prompt from `vc-follow-symlinks ask`  so `vc` fallbacks to `vc-follow-symlinks t` and since projects are usually under version control the value of `find-file-visit-truename` doesn't really meter.

I have another idea. I'll update PR.

Reply:
You are right. I think it's best to keep it simple and following vc here is the right choice. Unless you really have something uniformly better than this simple solution I would rather stick to this boolean setting.